packages/design-system/src/styles/components/_Button.scss

```diff
@@ -182,21 +181,26 @@
 %transparent-inverse-button {
   border-color: transparent;
   background-color: transparent;
-  // Default inverse focus styles are overridden above
-  @if $ds-include-focus-styles {
-    &:focus,
-    &.ds-c-button--focus {
-      border-color: transparent;
-    }
-  }
 
   &:disabled,
-  &:hover,
   &:active,
   &.ds-c-button--disabled,
-  &.ds-c-button--hover,
   &.ds-c-button--active {
     border-color: transparent;
+    background-color: transparent;
+  }
+  &:hover,
+  &.ds-c-button--hover {
+    color: rgba($color-base-inverse, 0.8);
```

Comment:
Needed unique hover styles for transparent inverse buttons

Reply:
I'm having trouble understanding why this change is necessary, or what it's accomplishing. Dont the transparent inverse buttons already have hover and disabled states?

Reply:
do you have a screenshot to compare the changes here?

Reply:
Ok, I see that this is probably feedback to DK's comment https://github.com/CMSgov/design-system/pull/946#pullrequestreview-596579401.

However, it would still help if you could walk me through why we need to add extra disabled and hover styles here

Reply:
After going back though this They aren't needed becasue they are already defined above, thanks!